lightkurve/lightcurvefile.py

```diff
@@ -97,10 +97,10 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
 
     @staticmethod
     def from_archive(target, cadence='long', quarter=None, month=None,
-                     campaign=None, **kwargs):
-        """Fetch a Light Curve File from the Kepler/K2 data archive at MAST.
+                     campaign=None, radius=1., targetlimit=1, verbose=True, **kwargs):
+        """Fetch a Lightcurve File from the Kepler/K2 data archive at MAST.
```

Comment:
Let's say `Fetch *one or more* Lightcurve files from ...` to make it clear that it can return multiple.